app/components/nypr-brand-header.js

```diff
@@ -0,0 +1 @@
+export { default } from 'nypr-ui/components/nypr-brand-header';
```

Comment:
can you put this in `app/components/nypr-ui/nypr-brand-header.js`? so when we use this it's `{{nypr-ui/brand-header}}`. that way you know where this component is coming from on templates.

Reply:
Oh are we moving things to that format and my lucky number got drawn to be the first?